solutions/cses-1196.mdx

```diff
@@ -2,14 +2,94 @@
 id: cses-1196
 source: CSES
 title: Flight Routes
-author: Benjamin Qi
+author: Aashish Patel, Benjamin Qi
 ---
+Solution 1:
+
+**Time Complexity:** $\mathcal{O}(mk\log (mk))$
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include<bits/stdc++.h>
+using namespace std;
+
+#define endl "\n"
+#define MOD 1000000007
+#define setprec(ans)  cout<<fixed<<std::setprecision(9)<<ans<<endl;
+typedef vector<int> VI;
+typedef vector<VI> VVI;
+typedef long long int ll;
+typedef vector<ll> VL;
+typedef vector<VL> VVL;
+typedef vector<pair<int, int>> VP;
+typedef vector<VP> VVP;
```

Comment:
usaco.guide coding conventions state that we need to use the following template:

```cpp
//BeginCodeSnip{C++ Short Template}
#include <bits/stdc++.h> // see /general/running-code-locally
using namespace std;

using ll = long long;

using vi = vector<int>;
#define pb push_back
#define all(x) begin(x), end(x)
#define sz(x) (int) (x).size()

using pi = pair<int,int>;
#define f first
#define s second
#define mp make_pair

void setIO(string name = "") {
	cin.tie(0)->sync_with_stdio(0); // see /general/fast-io
	if (sz(name)) {
		freopen((name + ".in").c_str(), "r", stdin); // see /general/input-output
		freopen((name + ".out").c_str(), "w", stdout);
	}
}
//EndCodeSnip

int main() {
  setIO();

}
```

@SansPapyrus683 do you want to detemplify this